### ISSUE TITLE
Column view gets stuck on "Processing..." when clicking on the root of a large codebase in the tree view

### ISSUE THREAD

**Issue by JonoYang** [nexB/aboutcode-manager#174]
This is happening on a sqlite DB that is 500+ MB.

**Comment by JonoYang** [comment]
I get this warning when clicking on items in the tree view:
```
/home/jono/nexb/tools/develop/aboutcode-manager/node_modules/sequelize/node_modules/bluebird/js/rel…:870 [31mUnhandled rejection RangeError: Maximum call stack size exceeded
    at Function.map (file:///home/jono/nexb/tools/develop/aboutcode-manager/assets/js/jquery-2.1.3.min.js:2:3574)
    at Function.mapToAttributeValues (file:///home/jono/nexb/tools/develop/aboutcode-manager/assets/js/aboutCodeBarChart.js:67:18)
    at Model.aboutCodeDB.FlattenedFile.findAll.then (file:///home/jono/nexb/tools/develop/aboutcode-manager/assets/js/aboutCodeBarChart.js:56:49)
    at Model.tryCatcher (/home/jono/nexb/tools/develop/aboutcode-manager/node_modules/sequelize/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/home/jono/nexb/tools/develop/aboutcode-manager/node_modules/sequelize/node_modules/bluebird/js/release/promise.js:510:31)
    at Promise._settlePromise (/home/jono/nexb/tools/develop/aboutcode-manager/node_modules/sequelize/node_modules/bluebird/js/release/promise.js:567:18)
    at Promise._settlePromise0 (/home/jono/nexb/tools/develop/aboutcode-manager/node_modules/sequelize/node_modules/bluebird/js/release/promise.js:612:10)
    at Promise._settlePromises (/home/jono/nexb/tools/develop/aboutcode-manager/node_modules/sequelize/node_modules/bluebird/js/release/promise.js:691:18)
    at Async._drainQueue (/home/jono/nexb/tools/develop/aboutcode-manager/node_modules/sequelize/node_modules/bluebird/js/release/async.js:133:16)
    at Async._drainQueues (/home/jono/nexb/tools/develop/aboutcode-manager/node_modules/sequelize/node_modules/bluebird/js/release/async.js:143:10)
    at Async.drainQueues (/home/jono/nexb/tools/develop/aboutcode-manager/node_modules/sequelize/node_modules/bluebird/js/release/async.js:17:14)[0m

printWarning @ /home/jono/nexb/tools/develop/aboutcode-manager/node_modules/sequelize/node_modules/bluebird/js/rel…:870
```

**Comment by jdaguil** [comment]
The issue is that the sqlite file was created with a previous version of Aboutcode Manager. In between those two versions the data schema changed. Since the sqlite file was first created with an AboutCode Manager that still used `dejacode_url` and the new table uses `reference_url`, the query will error. The DataTables message _"Processing..."_ is misleading since the query has been terminated with an error. The short fix is to use the same version of AboutCode Manager that you used to create the sqlite file. I believe that is ec7cf5e.

**Comment by jdaguil** [comment]
This can be closed. A crude fix has been done in #228 but will be addressed further in #230
